src/backend/distributed/citus--6.1-5--6.1-6.sql

```diff
@@ -0,0 +1,12 @@
+/* citus--6.1-5--6.1-6.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION get_shard_id_for_distribution_column(table_name regclass, distribution_value anyelement)
```

Comment:
I thought anyelement also contains arrays. We also have a test in this PR which deals with arrays. Besides when I try to create a function with any postgres gives me error saying "syntax error at or near any"